html/index.html.haml

```diff
@@ -426,7 +59,7 @@
               %a{:href => "http://machinery-project.org", :target => "_blank"}
                 Machinery
             .filter-input.col-md-3
-              %input#filter{"ng-model" => "query", :placeholder => "Type To Filter"}/
+              %input#filter{:placeholder => "Type To Filter"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace